opendatahub/scripts/manifests/fvt/fvt.yaml

```diff
@@ -53,7 +53,7 @@ spec:
             - /tmp/etcd.data
           # image: quay.io/coreos/etcd:v3.5.4
           # Tag -> registry.access.redhat.com/rhel7/etcd:3.2.32-34
-          image: registry.redhat.io/openshift4/ose-etcd@sha256:d3275cd886d13865937d225d8138db7f6b7bf59ac1a94d9fbe61e35286bee6ff
+          image: registry.redhat.io/openshift4/ose-etcd-rhel9@sha256:ea7545b79599f3868d442fdffdfe9b12a02a4b56ac155f02c0fac4720d475796
```

Comment:
I'm interested to know how this one is used, and why it can't be changed to `$(etcd)`. I guess it's because it's not kustomized in the same way? I wonder if maybe it should be?

Reply:
Yes, it is not kustomized in the same way. I could have added into the kustomization file the same override, but I noticed that there is a patch to delete the etcd configurations.

https://github.com/opendatahub-io/modelmesh-serving/blob/7e8b60b7d3486347242cfb15383f961240fb0b3b/opendatahub/scripts/manifests/fvt/kustomization.yaml#L5-L6

So, I thought it's no longer useful. I’d like to understand whether it makes sense to handle this here as well, or if it’s not really needed.

Reply:
this is only for test purposes, fvp on ci.

Reply:
Ok, thank you. What do you think could be the best way to manage it? Use the params.env as in config overlays, or keep as is?

Reply:
I noticed that fvt github actions does not run, there is an error and are never triggered. Is it known? Was it intentional?